src/main/java/ch/njol/skript/classes/data/JavaClasses.java

```diff
@@ -325,6 +326,36 @@ public String toVariableNameString(Quaternionf quaternion) {
 						return null;
 					}
 				}));
+
+		Classes.registerClass(new ClassInfo<>(UUID.class, "uuid")
+			.user("(uuid|universal(ly)? unique identifier)s?")
+			.name("UUID")
+			.description(
+				"A UUID is a universally unique identifier, which is a 128-bit number that is unique to each entity. "
+					+ "They are usually encoded as 36-character strings, which include 32 hexadecimal digits and "
+					+ "four hyphens, typically in this format: \"123e4567-e89b-12d3-a456-426614174000\"",
+				"In minecraft, it is used to identify entities (like players) in a way "
+					+ "that is unique across all entities, regardless of the entity's state.")
+			.since("INSERT VERSION")
+			.parser(new Parser<>() {
+				@Override
+				public @Nullable UUID parse(String string, ParseContext context) {
+					if (Utils.isValidUUID(string))
+						return UUID.fromString(string);
+					return null;
+				}
+
+				@Override
+				public String toString(UUID uuid, int flags) {
+					return "uuid \"" + uuid.toString() + "\"";
```

Comment:
Correct me if I'm wrong, but is this not a huge breaking change?
EX:
```vb
set {_u} to "%uuid of player%" #Why anyone would do this, I dunno, but it can happen
set {blah::%{_u%} to ... # No longer matches the variable save pattern
```
(I feel like theres probably other breaking changes here, but In my opinion, should just return as a toString
```suggestion
					return uuid.toString();
```

Reply:
I don't see many reasons why `uuid` shoud be prepended, we don't really need to know the type as it not only a few things that break it also breaks the `string value within (uuid of player) parsed as uuid`

Reply:
`string value within uuid` returns the actual uuid, without the prepended `uuid`
but yes, Shane is right. I should make it return just the toString